app/main.py

```diff
@@ -1,34 +1,57 @@
+from typing import Tuple, List, Union
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True):
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: Tuple[int],
```

Comment:
Tuple with one parameter?)
```suggestion
            start: Tuple[int, int],
```
Change in all places